packages/cli/src/commands/init/editTemplate.js

```diff
@@ -4,6 +4,8 @@ import path from 'path';
 import walk from '../../tools/walk';
 import {logger} from '@react-native-community/cli-tools';
 
+const DEFAULT_TITLE_PLACEHOLDER = 'Hello App Display Name';
```

Comment:
we'll be able to remove it once we add the title to default template, right?

Reply:
In theory - yes, but in practice, some of the templates might still use this string, so would keep it for a couple of versions. You can see that I typed `templateConfig` with optional type for `titlePlaceholder`.

Reply:
let's leave a TODO comment then

Reply:
Right C: